packages/api/src/external/hie/set-doc-query-progress-with-source.ts

```diff
@@ -108,18 +73,79 @@ export async function appendDocQueryProgressWithSource({
     return updatedPatient;
   });
 
-  await processDocQueryProgressWebhook({
+  await handleWebhookBeingSent({
```

Comment:
this one calls `handleWebhookBeingSent` w/ `await` but [the other call](https://github.com/metriport/metriport/pull/1538/files#diff-cda3d424b544bc28d9ff1595ff228c0d77328d38424e127d1b51fcf3d94859afR69) is not awaiting... shouldn't both be the same?